core/files/css/base.php

```diff
@@ -643,6 +643,34 @@ protected function is_update_required() {
 		return false;
 	}
 
+
+	/**
+	 * Clear stacks on additional breakpoints active.
+	 *
+	 * If a page contains templates, such as loop items, shortcodes, kits, or template widgets,
+	 * the templates responsive setting will not be updated unless the 'Regenerate CSS' option is used.
+	 * Reproduce only when additional breakpoints are active.
+	 *
+	 * @since 3.14.0
+	 *
+	 * @access private
+	 */
+	private function clear_stacks_on_additional_breakpoints_active() {
+		$is_edit_mode = Plugin::$instance->editor->is_edit_mode();
+		if ( ! $is_edit_mode && method_exists( Plugin::$instance->controls_manager, 'clear_stack_cache' ) ) {
```

Comment:
Why do you need to check if `method_exists`?

Reply:
Removed.